docs/installingNative.md

```diff
@@ -181,48 +194,21 @@ Most of these prerequisites can be installed using the package manager of your O
 
   You may need to restart the PostgreSQL server after changing the file above.
 
-- Run the test suite (Docker must be installed and running):
-
-  ```sh
-  make test
-  ```
-
-- Run the linters:
-
-  ```sh
-  make lint # or lint-js for Javascript only, or lint-python for Python only
-  ```
-
-- Create the file `PrairieLearn/config.json` with the path of your local course repository and with the path of a directory into which temporary files will be saved when using the in-browser file editor (edit both paths as needed):
-
-  ```json title="config.json"
-  {
-    "courseDirs": ["/Users/mwest/git/pl-tam212", "exampleCourse"],
-    "filesRoot": "../filesRoot",
-    "workspaceHostHomeDirRoot": "/tmp/workspace",
-    "workspaceHomeDirRoot": "/tmp/workspace"
-  }
-  ```
-
-- Run the server in development mode to automatically restart when changes are detected:
-
-  ```sh
-  make dev
-  ```
-
-  Alternatively, you can build and run the code to more closely mimic what will happen in production environments:
+## Configuration
 
-  ```sh
-  make build
-  make start
-  ```
+If you have your own [PrairieLearn course repository](./requestCourse/index.md), you will need to create the file `PrairieLearn/config.json` with the path of your local course repository. If you need support for workspaces, you should provide a path to a directory into which temporary files will be saved when using the in-browser file editor. Here is a sample configuration:
 
-- If you need support for [workspaces](workspaces/index.md), ensure Docker is installed and running, and then in a separate terminal run:
+```json title="config.json"
+{
+  "courseDirs": ["/Users/mwest/git/pl-tam212", "exampleCourse"],
+  "filesRoot": "../filesRoot",
+  "workspaceHostHomeDirRoot": "/tmp/workspace",
+  "workspaceHomeDirRoot": "/tmp/workspace"
+}
+```
 
-  ```sh
-  sudo make dev-workspace-host # or sudo make start-workspace-host
-  ```
+More information about the `config.json` can be found on the [server configuration](./configJson.md) page.
 
-- In a web-browser go to [http://localhost:3000](http://localhost:3000).
+## Development
 
-- To stop the server, use `Ctrl-C`.
+More information on the common commands and actions you do during development can be found on the [quickstart](./quickstart.md) page.
```

Comment:
```suggestion
More information on the development workflow can be found on the [quickstart](./quickstart.md) page.
```

Reply:
Changed in 11571eaf4cb629f088c608b49baefe0e835027bd.